client/my-sites/plugins/style.scss

```diff
@@ -362,3 +361,22 @@ $plugin-details-header-padding: 100px;
 		}
 	}
 }
+
+.plugin-details__sites-list-background + .plugin-details__body {
+	border-top: none;
+}
+
+.plugin-details__sites-list-background {
+	background-color: var( --studio-gray-0 );
+	padding-top: 20px;
+	padding-bottom: 60px;
+	position: relative;
+	transform: translateX(-50%);
+	left: 50%;
+	width: calc( 100vw - $sidebar-width );
```

Comment:
`$sidebar-width` doesn't work as expected here. It equals to `280px` when `--sidebar-width-max` is `272px` and `var( --sidebar-width-max )`  should be used. Otherwise it results to some white paddings
![SS 2021-11-19 at 12 16 51](https://user-images.githubusercontent.com/12430020/142606218-5062c134-9d25-46c2-bb88-557eb2cb001b.png)

To be more concise we should also subtract `1px`
```suggestion
	width: calc( 100vw - $sidebar-width );
```
```suggestion
	width: calc(100vw - var(--sidebar-width-max) -1px);
```

Reply:
and in that case we can omit `@import '@wordpress/base-styles/variables';`

Reply:
done: 36aeab24044da59454aabc46a6f36caa963c0809